app/utils/verenigingsregister.ts

```diff
@@ -287,3 +287,13 @@ type AttributeMap<T> = {
     | string
     | ((value: unknown, mapped: Record<string, unknown>) => void);
 };
+
+export function handleError(
+  toaster: { error: (message: string, title?: string) => void },
+  error: Error,
+) {
+  toaster.error(
+    'Er ging iets fout bij het verzenden van de gegevens naar het Verenigingsregister.',
```

Comment:
I just kept it simple for now (while still giving some extra information in the browser console).

Suggestions for a better message are welcome.

Reply:
If it's an http error, the API will provide more info though a JSON object. Perhaps the value of `response.detail` could be useful.